packages/main/src/plugin/kubernetes/contexts-states-dispatcher.ts

```diff
@@ -80,4 +83,12 @@ export class ContextsStatesDispatcher {
   getResourcesCount(): ResourceCount[] {
     return this.manager.getResourcesCount();
   }
+
+  updateResource(resourceName: string): void {
+    this.apiSender.send(`kubernetes-${resourceName}`);
+  }
+
+  getResources(resourceName: string): { contextName: string; items: readonly KubernetesObject[] }[] {
```

Comment:
nit: use a new interface/type there for the return type

Reply:
In fact it is `KubernetesContextResources`, changing